templates/cli/lib/commands/pull.js.twig

```diff
@@ -163,6 +193,14 @@ const pull = new Command("pull")
         helpWidth: process.stdout.columns || 80
     });
 
+pull
+    .command("all")
+    .description("Push all resource.")
+    .option(`--all`, `Flag to pull all functions`)
```

Comment:
```suggestion
    .option(`--all`, `Flag to pull all resources`)
```

Reply:
This `options` removed altogether, as it's handle globally